Client/src/Components/product-overview/StyleSelector.jsx

```diff
@@ -91,49 +92,69 @@ function StyleSelector({ setStyleIndex }) {
       </span>
       <div className={css.stylethumbs}>
         <div className={css.stylethumbsgrid}>
-          {styles.map(
-            (style, index) => (
-              <div className={css.thumbnail}>
-                {index === styleIndex
-              && (
-                <i className="far fa-check-circle" />
-              )}
-                <button type="button" className={css.stylebutton} onClick={() => (updateStyle(index))}>
-                  {" "}
-                  <img
-                    className={css.stylethumb}
-                    src={style.photos[0].thumbnail_url}
-                    alt={style.name}
-                    value={index}
-                  />
-                </button>
-              </div>
-            ),
-          )}
+          <RecordClicks widget="product overview" element="toggle style">
+            {styles.map(
+              (style, index) => (
+                <div className={css.thumbnail}>
+                  {index === styleIndex
+                && (
+                  <i className="far fa-check-circle" />
+                )}
+                  <button type="button" className={css.stylebutton} onClick={() => (updateStyle(index))} key={style}>
+                    {" "}
+                    <img
+                      className={css.stylethumb}
+                      src={style.photos[0].thumbnail_url}
+                      alt={style.name}
+                      value={index}
+                    />
+                  </button>
+                </div>
+              ),
+            )}
+          </RecordClicks>
         </div>
       </div>
       <form className={css.form}>
         <div className={css.formgrid}>
-          <select className={css.sizeselector} onChange={updateSelect}>
-            {sizes.map((size, index) => <option value={index} key={size.size}>{size.size}</option>)}
-          </select>
-          <select className={css.quantitySelector} onChange={updateQuantity}>
-            {sizes[currentSelect].quantity.map(
-              (qty) => (
-                <option
-                  value={qty}
-                  key={qty}
-                >
-                  {qty}
+          <RecordClicks widget="product overview" element="size selector">
+            <select className={css.sizeselector} onChange={updateSelect}>
+              {sizes.map((size, index) => (
+                <option value={index} key={size.size}>
+                  {size.size}
                 </option>
-              ),
-            )}
-          </select>
-          <button type="button" className={css.cartbutton} onClick={addToCart}>Add To Bag</button>
+              ))}
+            </select>
+          </RecordClicks>
+          <RecordClicks widget="product overview" element="quantity selector">
```

Comment:
RecordClick is only for onClick events, I don't know if it will work for  onChange events